test/tpu/xla_test_job.yaml

```diff
@@ -38,3 +38,21 @@ spec:
     env:
     - name: PJRT_DEVICE
       value: TPU
+  - name: xla-test-tpu-legacy
```

Comment:
hi @will-cromar , do you know any documentation showing me how to write the yaml file?

Reply:
See the Kubernetes documentation on Pods: https://kubernetes.io/docs/concepts/workloads/pods/

This will very likely fail, since it will run two containers concurrently that are going to try to open the TPU at the same time. Instead, add these scripts to the `xla-test` container and override the env vars as needed like `PJRT_DEVICE=TPU_LEGACY XLA_EXPERIMENTAL=... python3 /path/to/script`. Eventually we'll be able to remove both env vars and this should work by default

Reply:
Thanks. Fixed.